documentation/modules/managing/con-producer-config-properties.adoc

```diff
@@ -6,11 +6,18 @@
 = Kafka producer configuration tuning
 
 [role="_abstract"]
-Use a basic producer configuration with optional properties that are tailored to specific use cases.
-
+Use configuration properties to optimize the performance of Kafka producers.
+You can use standard Kafka producer configuration options.
 Adjusting your configuration to maximize throughput might increase latency or vice versa.
 You will need to experiment and tune your producer configuration to get the balance you need.
 
+When configuring a producer, consider the following aspects carefully, as they significantly impact its performance and behavior:
+
+Compression:: Implementing message compression can reduce network bandwidth usage, conserving resources and improving throughput.
```

Comment:
I guess you should clarify what resources does it conserve and what is the cost => network and disk storage migh be conserved, but CPU is the cost.